app/assets/stylesheets/layout_components/credit-card.scss

```diff
@@ -0,0 +1,37 @@
+.credit-card{
+  background-color: $cerulean;
+  border-radius: 3px;
+  padding: $base-padding;
+  max-width: 400px;
+  min-height: 216px;
+  position: relative;
+
+  &__logo{
+    background-size: contain;
+    height: 50px;
+    width: 100px;
+    float: right;
+
+    &--visa{
+      @extend .credit-card__logo;
+      background: url(image-path("cc-type-visa-white.png")) no-repeat;
+      width: 100px;
+    }
+
+    &--mc{
+      @extend .credit-card__logo;
+      background: url(image-path("cc-type-mc-white.png")) no-repeat;
```

Comment:
this logo doesn't look right, it looks like the stroke on the circles is too small? 
![image](https://cloud.githubusercontent.com/assets/1906696/23811121/d2023b34-0591-11e7-8f02-5faf6fd991f3.png)
![image](https://cloud.githubusercontent.com/assets/1906696/23811123/d3b03094-0591-11e7-8d79-ae314b9f2fe5.png)